src/components/EventDateFilters.vue

```diff
@@ -1,62 +1,92 @@
 <template>
-  <div>
+  <div class='event-date-filters'>
     <h5 v-if="showTitle" class="filter-events-title">Date of event</h5>
     <div class="filter-events-date-picker">
-      <input ref="startDate" type="text" :value="startDate" />
-      <strong>to</strong>
-      <input ref="endDate" type="text" :value="endDate" />
+      <div class='date-picker-header'>
+        <input
+          autofocus
+          v-on:click="currentCalendar = 'startDate'"
+          ref="startDate"
+          type="text"
+          :value="startDate"
+        />
+        <strong>to</strong>
+        <input
+          v-on:click="currentCalendar = 'endDate'"
+          ref="endDate"
+          type="text"
+          :value="endDate"
+        />
+      </div>
+      <div id='calendar-container'>
+        <div v-show="currentCalendar === 'startDate'" id='start-calendar' />
+        <div v-show="currentCalendar === 'endDate'" id='end-calendar' />
+      </div>
     </div>
   </div>
 </template>
 
 <script>
-import moment from 'moment';
-
-// sigh
-window.moment = moment;
 
+import moment from 'moment';
 import Pikaday from 'pikaday';
 
 // e.g., 2017-03-01
 const inputFormat = 'YYYY-MM-DD';
+
 // e.g., Mar 1, 2017
-const displayFormat = 'MMM D, YYYY'
+const displayFormat = 'MMM D, YYYY';
 const forDisplay = (date) => {
-  if(date){
+  if (date) {
     return moment(date, inputFormat).format(displayFormat);
-  }else{
-    return null
+  } else {
+    return null;
   }
 }
 
+// Pikaday needs all of these i18n values just to override the weekdays to
+// be single letters.  Everything is default except `weekdaysShort`
+const calendarFormat = {
+  previousMonth: 'Previous Month',
+  nextMonth: 'Next Month',
+  months: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+  weekdays: ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday'],
+  weekdaysShort: ['S', 'M', 'T', 'W', 'T', 'F', 'S']
+};
+
 export default {
   name: 'event-date-filters',
   props: ['filters', 'showTitle'],
+  data() {
+    return { currentCalendar: 'startDate' };
+  },
   computed: {
     startDate: {
-      get(){
+      get() {
         return forDisplay(this.$store.state.filters.startDate);
       },
-      set(value){
-        this.$store.commit('setFilters',{startDate: value})
+      set(value) {
+        this.$store.commit('setFilters',{ startDate: value });
       }
     },
     endDate: {
       get(){
         return forDisplay(this.$store.state.filters.endDate);
       },
       set(value){
-        this.$store.commit('setFilters',{endDate: value})
+        this.$store.commit('setFilters',{ endDate: value })
       }
     }
   },
 	mounted() {
     const component = this;
 
     this.pikadayStartDate = new Pikaday({
-      field: this.$refs.startDate,
+      field: component.$refs.startDate,
       format: displayFormat,
-      position: 'bottom',
+      bound: false,
```

Comment:
Based on the mobile design:

![image](https://cloud.githubusercontent.com/assets/1724544/24642116/c8989eea-18b9-11e7-8cae-2ec824dab75a.png)

As well as the desktop one from the AI file, I think the idea for both cases might be for the date picker calendar to be initially not visible, then on input focus, it joins the party. For desktop I think it looks great as-is in this PR, with the calendar immediately showing up in the dropdown, but it might be easier to have it work out of the box for both mobile and desktop if it starts out not visible.....however you think it works best tho, +1.